src/views/app/group-head-summary/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { ParamListBase, useNavigation } from '@react-navigation/native';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React from 'react';
+import ViewWrapper from '../../../components/layout/viewWrapper';
+import useScreenFocus from '../../../hooks/focus';
+
+const CampusGroupHeads: React.FC<NativeStackScreenProps<ParamListBase>> = ({ navigation, route }) => {
+    const { navigate } = useNavigation();
```

Comment:
Same as earlier, you don't need this hook here. deconstruct `navigation` from `props`.